src/main/scala/cromwell/engine/RuntimeAttribute.scala

```diff
@@ -0,0 +1,9 @@
+package cromwell.engine
+
+object RuntimeAttribute extends Enumeration {
+  type RuntimeAttribute = Value
+  val failOnStderr = Value("failOnStderr")
+  val docker = Value("docker")
+  val serialize = Value("serialize")
+  val memory = Value("memory")
+}
```

Comment:
I'll remove this enum since its only usage was within the aforementioned package.scala.